packages/remote-feature-flag-controller/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@metamask/remote-feature-flag-controller",
+  "version": "0.0.0",
+  "description": "Controller with caching, fallback, and privacy for managing feature flags via ClientConfigAPI",
```

Comment:
Similar to the README, I don't find this to be a clear description

Reply:
Updated in d2711d70be092faf2349c10b1ac1de4b8cbad975